tests/pds/dataservice_test.go

```diff
@@ -627,6 +628,88 @@ var _ = Describe("{UpgradeDataServiceImage}", func() {
 	})
 })
 
+var _ = Describe("{CordonNodeAndDeletePod}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("CordonNodeAndDeletePod", "Deploys a data service, cordons one selected node, deletes the pod from that node", pdsLabels, 0)
+	})
+
+	It("Cordon a Node and Delete Pods from that node", func() {
+		Step("Deploy, Validate, Cordon Node and Delete Pods, Validate new Pod, Validate Storage, Run Workload on Data Service", func() {
+			for _, ds := range params.DataServiceToTest {
+				// Run this Test on Consul only. We can enable more Data Services later from here if needed.
+				// Consul is a random choice. We can choose any other Data service as well.
+				// This test case is pretty much Data service independent
+				if ds.Name != consul {
+					log.Warnf("Skipping %v dataservice ", ds.Name)
+					continue
+				}
+				isDeploymentsDeleted = false
+				deployment, _, _, err = DeployandValidateDataServices(ds, tenantID, projectID)
+				log.FailOnError(err, "Error while deploying data services")
```

Comment:
can you add the name of dataservice in the error log

Reply:
Done